modular_skyrat/modules/customization/modules/mob/living/silicon/topic.dm

```diff
@@ -5,3 +5,13 @@
 		examine_panel.ui_interact(usr) //datum has a tgui component, here we open the window
 	if(href_list["temporary_flavor"]) // we need this here because tg code doesnt call parent in /mob/living/silicon/Topic()
 		show_temp_ftext(usr)
+//bubber edit begin
+	if(href_list["open_door"])
+		var/obj/machinery/door/airlock/door = locate(href_list["open_door"]) in SSmachines.get_machines_by_type_and_subtypes(/obj/machinery/door/airlock)
+		var/mob/living/target = locate(href_list["user"]) in GLOB.mob_list
+		if(!target)
+			return
+		if(!door)
+			return
+		open_door(target, door)
```

Comment:
What is target here? Also, you need to sanity check that the AI can actually see the door here- the href  can have arbitrary contents(this is data from the client- a modified client can put whatever it wants in there!